### ISSUE TITLE
screenshoting 

### ISSUE THREAD

**Issue by mal1ceee** [DarthAffe/ScreenCapture.NET#24]
hi there, i was wondering if there is any implementation of this that takes in the window handle and screenshot the provided window?

**Comment by DarthAffe** [comment]
No, all data is copied out of the front-buffer. It's not possible to capture specific windows. (Aside from capturing a region using the boundries of the window.)

**Comment by mal1ceee** [comment]
So theoretically I could screenshot a specific window given the boundaries?

**Comment by DarthAffe** [comment]
As long as it is in foreground yes. You'd basically take a image of your screen and cut that region out.

**Comment by mal1ceee** [comment]
ah thanks :)
